src/views/partials/doc-resource-subclasses.ejs

```diff
@@ -15,7 +14,128 @@
   "desc": [
     "The archetypal Champion focuses on the development of raw physical power honed to deadly perfection. Those who model themselves on this archetype combine rigorous training with physical excellence to deal devastating blows."
   ],
-  "features": [],
+  "spells": [
```

Comment:
Should we switch this to the Paladin subclass instead so it's consistent?

Reply:
I have swapped the example to be consistent